src/PageHeader/manageResponsiveValueWithCSS.ts

```diff
@@ -0,0 +1,53 @@
+import {isResponsiveValue, ResponsiveValue} from '../hooks/useResponsiveValue'
+import {BetterSystemStyleObject} from '../sx'
+
+/**
+ * This function is inspired by the `useResponsiveValue` hook and it's used to render responsive values with CSS.
+ * @param value - The value that we want to render responsively
+ * @param fallback - The fallback value
+ * @param cssProperty - The CSS property that we want to render
+ * @param truety - The value that we want to render when the value is true
+ * @param falsy - The value that we want to render when the value is false
+ 
+ * @example
+ * CSSManagedResponsiveValue({narrow: true, regular: true, wide: false}, false, 'display', 'none', 'flex')
+ * @returns
+ * {
+ *   "@media screen and (max-width: 768px)": {
+ *     "display": "none"
+ *   },
+ *   "@media screen and (min-width: 768px)": {
+ *     "display": "none"
+ *   },
+ *   "@media screen and (min-width: 1440px)": {
+ *     "display": "flex"
+ *   }
+ * }
+ */
+export function CSSManagedResponsiveValue<T, F, B, C>(
+  value: T,
+  fallback: F,
+  cssProperty: string,
+  truety: B,
+  falsy: C,
```

Comment:
```suggestion
  truthy: B,
  falsey: C,
```

Small nit for the truth-y and false-y names. Another idea for names could be `enabled` and `disabled` for if the responsive value is enabled or disabled at that breakpoint

Reply:
Sorry! 🙈 I changed the implementation a little bit (I explained it as a response here), so we don't need these anymore.